src/prefect/logging/handlers.py

```diff
@@ -208,6 +208,9 @@ def prepare(self, record: logging.LogRecord) -> Dict[str, Any]:
                     "Encountered malformed run context. Does not contain flow or task "
                     "run information."
                 )
+        # allow logger to be used without a known flow run id
+        if flow_run_id in ("None", "unknown"):
```

Comment:
any reason the other "unknown" reference has brackets and this one doesn't?

Reply:
nice catch!

I have now removed this because AFAICT this concerns the capture / sending of logs to the API, not explicitly `get_run_logger` - will bring this change back with an explanation on the `in ("None", "unknown")` check in a later PR